packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [14.0.0]
+### Added
```

Comment:
Do we need to include the changelogs from https://github.com/MetaMask/core/pull/1725?

Reply:
oh lol theyre there whoops